shapash/explainer/smart_plotter.py

```diff
@@ -452,15 +461,17 @@ def plot_violin(self,
 
             else:
                 fig.add_trace(go.Violin(x=feature_values.loc[feature_values.iloc[:, 0] == i].values.flatten(),
-                                        y=contributions.loc[feature_values.iloc[:, 0] == i].values.flatten(),
+                                        y=contributions.loc[feature_values.iloc[:, 0]
+                                                            == i].values.flatten(),
```

Comment:
the code become difficult to read

Reply:
The question is do we respect rules or not ? what is the guideline to follow ?

Reply:
I tried to fix it

Reply:
generate a loss of clarity